DfE.FindInformationAcademiesTrusts/Extensions/CategoriesOfConcernExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿using DfE.FindInformationAcademiesTrusts.Data;
+
+namespace DfE.FindInformationAcademiesTrusts.Extensions;
+
+public static class CategoriesOfConcernExtensions
+{
+    public static string ToDataSortValue(this CategoriesOfConcern rating)
+    {
+        return rating.ToDisplayString().ToLowerInvariant().Trim();
+    }
+
+    public static string ToDisplayString(this CategoriesOfConcern rating)
+    {
+        return rating switch
+        {
+            CategoriesOfConcern.None => "None",
+            CategoriesOfConcern.SpecialMeasures => "Special Measures",
+            CategoriesOfConcern.SeriousWeakness => "Serious Weakness",
+            CategoriesOfConcern.NoticeToImprove => "Notice To Improve",
+            _ => "Unknown"
```

Comment:
From old PR: https://github.com/DFE-Digital/find-information-about-academies-and-trusts/pull/634#discussion_r1860992718
![image](https://github.com/user-attachments/assets/6cfe38fd-c747-4795-85c0-5b3176787ea7)